crates/bitwarden-napi/src-ts/bitwarden_client/schemas.ts

```diff
@@ -254,7 +254,11 @@ export interface SecretCreateRequest {
      * Organization where the secret will be created
      */
     organizationId: string;
-    value:          string;
+    /**
+     * IDs of the projects that this secret will belong to
+     */
+    projectIds?: string[] | null;
```

Comment:
@Hinton thanks for pointing this out. I spoke with @patrick-bitwarden and @dani-garcia. We want to allow the ability to pass in multiple project ids via the bws cli, though it's not supported _yet_ in the backend. Patrick recommended we show a more friendly error instead of one from the server, so I created an internal ticket for it.